core/src/test/java/org/elasticsearch/rest/RestControllerTests.java

```diff
@@ -34,9 +35,13 @@
 import java.util.concurrent.TimeUnit;
 
 import static org.hamcrest.CoreMatchers.equalTo;
+import static org.mockito.Matchers.any;
+import static org.mockito.Matchers.eq;
+import static org.mockito.Mockito.doCallRealMethod;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
 
 public class RestControllerTests extends ESTestCase {
-
```

Comment:
I think we prefer the extra blank line between the definition of the class and the body of the class (and between the body of the class and the closing brace for the class). No, this isn't held to perfectly, but you'll see it across the codebase.